packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -614,6 +614,7 @@ function findLocalizePositions(
     ast = parseSync(options.code, {
       babelrc: false,
       sourceType: 'script',
+      filename: options.filename,
```

Comment:
Can you add `configFile: false,` here as well to ensure configuration isolation.

Reply:
🙏

Reply:
Actually, that should be added in several other places as well.  I'll create a separate PR for it.